docs/contents/components/forms/hue-slider/accessibility.ja.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: HueSlider
+description: "`HueSlider`は、ユーザーが色相を選択するために使用されるコンポーネントです。"
+package_name: "@yamada-ui/color-picker"
+order: 4
+with_description: true
+tab: アクセシビリティ
+---
+
+`HueSlider`に`aria-label`を設定すると、スクリーンリーダーによって読み上げられます。
+
+```tsx live=false
+<HueSlider aria-label="Hue slider" />
+```
+
+## キーボード操作
+
+| キー         | 説明                                            | 状態 |
+| ------------ | ----------------------------------------------- | ---- |
+| `ArrowDown`  | スライダーの値を1ステップ分減少します。         | -    |
+| `ArrowLeft`  | スライダーの値を1ステップ分減少します。         | -    |
+| `ArrowRight` | スライダーの値を1ステップ分増加します。         | -    |
+| `ArrowUp`    | スライダーの値を1ステップ分増加します。         | -    |
+| `End`        | スライダーの最大値に移動します。                | -    |
+| `Home`       | スライダーの最小値に移動します。                | -    |
+| `PageDown`   | スライダーの値を大きく減少(全体の10%分)します。 | -    |
+| `PageUp`     | スライダーの値を大きく増加(全体の10%分)します。 | -    |
+
+## ARIAロールと属性
+
+| コンポーネント | ロールと属性    | 使い方                                                                                                        |
+| -------------- | --------------- | ------------------------------------------------------------------------------------------------------------- |
+| Slider         | `role="slider"` | スライダーであることを示します。                                                                              |
```

Comment:
Change `Slider` to `SliderThumb`.
`SliderThumb` will separate by https://github.com/yamada-ui/yamada-ui/issues/3338.

Reply:
You probably misunderstand.
You don't neet to change role. Please change `Slider` of `Component`.